config.nims

```diff
@@ -11,4 +14,22 @@ else:
 
 # Workaround for NilAccessError crash on Windows #98
 when defined(Windows):
-  switch("gc", "markAndSweep")
\ No newline at end of file
+  switch("gc", "markAndSweep")
+
+const workaround_10629 = defined(windows) and getCommand() == "doc"
+  # pending https://github.com/nim-lang/Nim/pull/10629
+when workaround_10629:
+  switch("define", "workaround_10629")
+  import ospaths, strformat, strutils
+  proc getNimRootDir(): string =
+    fmt"{currentSourcePath}".parentDir.parentDir.parentDir
+  var file = getNimRootDir() / "lib/pure/includes/osseps.nim"
+  if not existsFile file:
+    file = getNimRootDir() / "lib/pure/ospaths.nim"
+  const file2 = getTempDir() / "ospaths.nim"
+  var text = file.readFile
+  text = replace(text, """
+    DirSep* = '/'""", """
+    DirSep* = '\\'""")
+  writeFile file2, text
+  patchFile("stdlib", file.splitFile.name, file2)
```

Comment:
Still don't understand why we need all this. Doesn't look like the PR was accepted so I'm concerned this will be here for good.

nim doc is working locally, at least on Windows, without this code. Can we skip it?